business_requirement_deliverable_cost/models/business.py

```diff
@@ -185,6 +185,29 @@ class BusinessRequirement(models.Model):
         'RL Total Cost',
         compute='_compute_rl_total_cost'
     )
+    total_revenue_ci = fields.Float(
+        compute='_compute_total_revenue_in_ci',
+        string='Total Revenue in CI',
+    )
+    ci_currency_id = fields.Many2one(
+        comodel_name='res.currency',
+        string='Currency',
+        readonly=True,
+        compute='_compute_get_ci_currency'
+    )
+
+    @api.multi
+    def _compute_get_ci_currency(self):
+        for br in self:
+            br.ci_currency_id = self.env.user.company_id.currency_id.id
+
+    @api.depends('total_revenue')
```

Comment:
@YogeshMahera-SerpentCS should be depend on this `currency_id` field so if change will be computed.